sdk/python/feast/feature_view_projection.py

```diff
@@ -10,15 +10,26 @@
 
 @dataclass
 class FeatureViewProjection:
+    """
+    A feature view projection represents a set of features from a single feature view.
+
+    Attributes:
+        name: The unique name of the feature view projection.
```

Comment:
Wait, but feature view projections aren't public facing objects right? So `name` is really the aliased name of the feature view, not the projection. Or am I missing something?